src/mini_s3.erl

```diff
@@ -21,91 +21,107 @@
 
 -module(mini_s3).
 
--export([new/2,
-         new/3,
-         new/4,
-         new/5,
+-export([
+         copy_object/5,
+         copy_object/6,
          create_bucket/3,
          create_bucket/4,
          delete_bucket/1,
          delete_bucket/2,
-         get_bucket_attribute/2,
-         get_bucket_attribute/3,
-         list_buckets/0,
-         list_buckets/1,
-         set_bucket_attribute/3,
-         set_bucket_attribute/4,
-         list_objects/2,
-         list_objects/3,
-         list_object_versions/2,
-         list_object_versions/3,
-         copy_object/5,
-         copy_object/6,
          delete_object/2,
          delete_object/3,
          delete_object_version/3,
          delete_object_version/4,
+         get_bucket_attribute/2,
+         get_bucket_attribute/3,
          get_object/3,
          get_object/4,
          get_object_acl/2,
          get_object_acl/3,
          get_object_acl/4,
          get_object_torrent/2,
          get_object_torrent/3,
-         get_object_metadata/3,
          get_object_metadata/4,
-         s3_url/6,
+         get_host_toggleport/2,
+         list_buckets/1,
+         list_objects/2,
+         list_objects/3,
+         list_object_versions/2,
+         list_object_versions/3,
+         new/3,
+         new/4,
+         new/5,
          put_object/5,
          put_object/6,
+         s3_url/6,
+         s3_url/7,
+         set_bucket_attribute/3,
+         set_bucket_attribute/4,
          set_object_acl/3,
-         set_object_acl/4]).
+         set_object_acl/4
+]).
 
--export([manual_start/0,
-         make_authorization/10,
-         make_signed_url_authorization/5,
-         universaltime/0]).
-
--ifdef(TEST).
--compile([export_all]).
--include_lib("eunit/include/eunit.hrl").
--endif.
+-export([make_authorization/10,
+         manual_start/0,
+         make_expire_win/2,
+         universaltime/0
+]).
 
+% is this used?  TODO: try removing
 -include("internal.hrl").
+%-include_lib("eunit/include/eunit.hrl").  % <----- need to delete this
```

Comment:
Can we clean up this commented code? What do we need to figure out here?

Reply:
Fixed.